.github/workflows/maven.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
```

Comment:
Does this belong in master? There have been quite a few hiccups with it, I don't think we know how it works exactly